### DISCUSSION TITLE
[Q] Where is the Dist folder?

### DISCUSSION THREAD

**Discussion by Bobbygllh** [exceljs/exceljs#2389] [Q&A]
The readme mentions a dist folder in the root directory but I have been unable to locate the folder

> The ES5 code is also browserified and available as dist/exceljs.js or dist/exceljs.min.js

I am looking for the exceljs.min.js

thank you!

**Comment by cesarjhony** [comment]
Hi Bobbygllh, 

The people is using npm/node for all, then to get the dist folder you need to create an any npm project and install exceljs
`npm install exceljs`
So open node_modules/exceljs folder and voila, dist is on there, just copy the file that you need to lib of your project.

Today I was the same doubt, I think it is essential that information on the readme content.
